db/create/create_table_pokemon.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE `pokemon` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `name` varchar(45) NOT NULL,
+  `size` int DEFAULT NULL,
+  `type1Id` int NOT NULL,
+  `type2Id` int DEFAULT NULL,
+  `nationalDexNumber` int NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `name` (`name`),
+  UNIQUE KEY `nationalDexNumber` (`nationalDexNumber`),
+  KEY `type1Id` (`type1Id`),
+  KEY `type2Id` (`type2Id`),
+  CONSTRAINT `pokemon2_ibfk_1` FOREIGN KEY (`type1Id`) REFERENCES `type` (`id`),
```

Comment:
Typo on the constraint name. It should be `pokemon_ibfk_1`. The same applies to the next one.